### DISCUSSION TITLE
Warning: Tooltip: `key` is not a prop.

### DISCUSSION THREAD

**Discussion by ricardo-fnd** [ReactTooltip/react-tooltip#1089] [Q&A] [answered]
I'm getting this error, when I pass `key` as a prop
![Screenshot 2023-09-11 at 18 01 55](https://github.com/ReactTooltip/react-tooltip/assets/52207061/59786703-ffa4-4729-aabb-0d14334fc02f)

And actually, looking for the arguments that `<Tooltip />` accepts, I don't see a `key` prop there:
![Screenshot 2023-09-11 at 18 00 59](https://github.com/ReactTooltip/react-tooltip/assets/52207061/05664ef1-2553-4736-af79-95dccfecc5b7)

**Comment by gabrieljablonski** [comment, marked as answer]
Hi Ricardo. Unfortunately there's not enough information here to be able to help you for sure.

As you've noticed, the tooltip component indeed does not have an explicit `key` prop, but since it is a functional component, the `key` prop is implicit. Here's [an example](https://legacy.reactjs.org/docs/lists-and-keys.html#:~:text=%3CListItem%20key%3D%7Bnumber.toString()%7D%20value%3D%7Bnumber%7D%20/%3E) from the official React docs.

From the error message, it seems like you're defining a wrapper component for the tooltip on your `components13/Tooltip.ts` file.

My guess is that, as the error message suggests, you're trying to expose the `key` prop, which is not allowed. If that's not the case, please provide more information.

**Reply by ricardo-fnd** [reply]
That is the case, yes. I shouldn't accept `key` prop, as you said, its implicit for react components. thank you so much for the help
